rc.local.copyme

```diff
@@ -0,0 +1 @@
+python3 signal_strength_tracker.py &
```

Comment:
Is the user home directory the default for `rc.local` ? I recommend an absolute path.

Also, virtual environment?